src/Components/Web.JS/package.json

```diff
@@ -29,13 +29,16 @@
     "ts-loader": "^4.4.1",
     "typescript": "^3.8.3",
     "webpack": "^4.42.1",
-    "webpack-cli": "^3.3.11"
+    "webpack-cli": "^3.3.11",
+    "yarn-audit-fix": "^9.2.2"
```

Comment:
Do not add this dependency. `yarn-audit-fix` should be in your global cache but not listed as a dev dependency in this repo.